packages/kotti-ui/source/kotti-popover/KtPopoverIconTextItem.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+	<div
+		class="kt-popover-item"
+		:class="{
+			'kt-popover-item--is-disabled': isDisabled,
+		}"
+		v-on="$listeners"
+	>
+		<i v-show="icon" class="yoco" v-text="icon" />
+		<div>{{ label }}</div>
+	</div>
+</template>
+
+<script>
+export default {
+	name: 'KtPopoverIconTextItem',
+	props: {
+		icon: {
+			type: String,
+			default: null,
+		},
+		isDisabled: {
+			type: Boolean,
+		},
+		label: {
+			type: String,
+		},
+	},
+}
+</script>
+
+<style lang="scss">
+@import '../kotti-style/_variables.scss';
+.kt-popover-item {
+	display: flex;
+	padding: $unit-2;
+	margin: $unit-h;
+	line-height: 20px;
+	&:hover {
+		cursor: pointer;
+		background: $lightgray-300;
```

Comment:
for the colors specially, we r moving towards using css variables now 
see https://3yourmind.github.io/kotti/tokens/introduction and check how it's used in other places of `kotti-ui` for references

I'd propose you contact the designers for this b/c i'm always confused as to what should be the color for  certain things

Reply:
https://3yourmind.github.io/kotti/tokens/colors see this also